src/assets/CDN_IMAGES.ts

```diff
@@ -0,0 +1,21 @@
+const CDN_BASE = import.meta.env.VITE_CLOUDINARY_BASE_URL;
```

Comment:
move these to constants (different constants for API base URL and the paths itself)